content/source/docs/enterprise/private/config.html.md

```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
 In all examples below, be sure to replace "`<TFE HOSTNAME>`" with the hostname
 of your Private Terraform Enterprise instance.
 
-Navigate to `https://<TFE HOSTNAME>/admin/bootstrap` in your browser. You will
-be presented with a status page that looks like this:
+Navigate to `https://<TFE HOSTNAME>:8800/` in your browser. You will
+be presented with the installer's dashboard that looks like this:
 
-![Bootstrap Status Page](./assets/bootstrap-status.png)
+![PTFE Installer Dashboard](./assets/post-install-repl-console.png)
 
 ### Creating an Administrator
 
-After clicking on "Begin Installation" you will be brought to a page called
-"Create Administrator". Enter the requested information to create the first
-Administrator account. You will be able to create additional administrators
-later.
+After clicking on the "Open", right below the "Stop Now" button, you will
+be brought to a page asking you to create the first PTFE administrator account.
+You will be able to create additional administrators once you login.
+
+![New admin dialog](./assets/create-an-account.png)
 
 ### Creating an Organization
 
 The next step will create the first Organization.
 
 ![Create a new organization](./assets/create-organization.png)
 
-If you skip this step, you will be prompted the create the first organization
-when you first log in.
```

Comment:
Does this not apply anymore?

Reply:
There is no skip button. If you navigate elsewhere (user prefs, admin settings, whatever) and then click back on the TFE icon you'll be asked to create an org. If you log out and log in again, you'll be asked to create an org.

Reply:
Oh, good call. I forgot that you're already logged in with this new flow.